bin/ansible-connection

```diff
@@ -189,69 +140,112 @@ class Server():
 
                     signal.alarm(self.play_context.timeout)
```

Comment:
missing alarm timer reset at conclusion of method run... should be somewhere around line 156